examples/rl/workflows/policy_host.py

```diff
@@ -16,8 +16,13 @@
 
 if __name__ == "__main__":
```

Comment:
I am still worried about whether it is necessary to maintain `policy host` and whether it will cause too many times of data transfers.

I remember that it is used for the situation where there are too many policies for one single node to hold?

@ysqyang  @buptchan

Reply:
I remember @ysqyang  said it's necessary because too many policies, e.g., 1000 policies, can cause `out of memory` in `policy_manager`.

Reply:
Yes, this is paired with DistributedPolicyManager where the policy instances need to be distributed across several nodes, i.e., hosts.